src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -554,40 +585,51 @@
      reaction-picker]))
 
 (defmethod ->message constants/content-type-image
-  [{:keys [content in-popover? outgoing] :as message}
+  [{:keys [content community? community-id can-manage-users? from in-popover? outgoing] :as message}
    {:keys [on-long-press modal ref]
     :as   reaction-picker}]
   (let [on-long-press (fn []
                         (on-long-press
                          (concat [{:type     :main
                                    :on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
                                    :id       :reply
-                                   :icon     :main-icons2/reply-context20
+                                   :icon     :main-icons2/reply
                                    :label    (i18n/label :t/message-reply)}
                                   {:type     :main
                                    :on-press #(re-frame/dispatch [:chat.ui/save-image-to-gallery (:image content)])
                                    :id       :save
-                                   :icon     :main-icons2/save-context20
+                                   :icon     :main-icons2/save
                                    :label    (i18n/label :t/save-image-library)}
                                   {:type     :main
                                    :on-press #(images/download-image-http
                                                (get-in message [:content :image]) preview/share)
                                    :id       :share
-                                   :icon     :main-icons2/share-context20
+                                   :icon     :main-icons2/share
                                    :label    (i18n/label :t/share-image)}]
                                  [{:type     :danger
                                    :on-press #(re-frame/dispatch
                                                [:chat.ui/delete-message-for-me message
                                                 config/delete-message-for-me-undo-time-limit-ms])
                                    :label    (i18n/label :t/delete-for-me)
-                                   :icon     :main-icons2/delete-context20
+                                   :icon     :main-icons2/delete
                                    :id       :delete-for-me}]
                                  (when (and outgoing config/delete-message-enabled?)
                                    [{:type     :danger
                                      :on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
                                      :label    (i18n/label :t/delete-for-everyone)
-                                     :icon     :main-icons2/delete-context20
-                                     :id       :delete}]))))]
+                                     :icon     :main-icons2/delete
+                                     :id       :delete}])
+                                 (when (and community? can-manage-users?)
```

Comment:
This logical check is repeated a couple times throughout this file, but it has an important meaning. How about we extract it into a function named `allow-dangerous-community-actions?` (or any other meaningful name). As @du64 mentioned in https://github.com/status-im/status-mobile/pull/14276#issuecomment-1296345570, this function could also take care of checking community owners cannot be kicked/banned.

Reply:
Hey Miloš!

Thanks for taking the effort to refactor. Unfortunately your questions are too specific for me, because I haven't been working in the communities epic. I'm looping in other devs to see if they can help.

@cammellos, @jo-mut, @J-Son89, perhaps one of you can help @mrdaak reliably detect if the sender is a community group owner/admin, so that we can avoid banning/kicking them.

Reply:
I'd say not to worry about kicking banning the owner, he has full control of the community, so even if it's kicked/banned, he's able to join it back. Also it would not impact the management as that's done through the community keypair, so it should be fine, but we can look more into it in a separate PR, I think for this PR, trusting `can-manage-users?` is enough